routers/user/profile.go

```diff
@@ -97,6 +98,10 @@ func Profile(ctx *context.Context) {
 	// so everyone would get the same empty heatmap
 	ctx.Data["EnableHeatmap"] = setting.Service.EnableUserHeatmap && !ctxUser.KeepActivityPrivate
 	ctx.Data["HeatmapUser"] = ctxUser.Name
+	if ctx.User != nil && len(ctx.User.Description) != 0 {
```

Comment:
So user must login to see the profile?
`ctx.User` should the login user, `cxtuser` is the user who has the profile?

Reply:
Good catch @lunny sorry I missed that